i18n-scripts/.eslintrc.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  root: true,
+  extends: ['plugin:console/node-typescript-prettier'],
+  rules: {
+    'no-console': 'off',
+    // fs.promises requires a newer version of node however our compliance is set to node >=10
```

Comment:
Node 10 is no longer supported. I'd recommend targeting a newer version and updating the script. (We need to do this in the console repo as well.)

Reply:
removed